subsys/fb/cfb.c

```diff
@@ -477,7 +477,7 @@ int cfb_framebuffer_finalize(const struct device *dev)
 		.pitch = fb->x_res,
 	};
 
-	if (!(fb->pixel_format & PIXEL_FORMAT_MONO10) != !(fb->inverted)) {
+	if (!(fb->pixel_format & PIXEL_FORMAT_MONO10) == !(fb->inverted)) {
```

Comment:
Can be simplified to:
```
if ((fb->pixel_format & PIXEL_FORMAT_MONO10) == fb->inverted) {
```

Reply:
I'm pretty sure that doesn't work since `fb->pixel_format & PIXEL_FORMAT_MONO10` is an int while `fb->inverted` is a bool, but I did change it to be more simple. What do you think?